core/src/main/java/hudson/util/FormFieldValidator.java

```diff
@@ -145,6 +146,7 @@ public final void process() throws IOException, ServletException {
     /**
      * Gets the parameter as a file.
      */
+    @SuppressFBWarnings(value = "PATH_TRAVERSAL_IN", justification = "Not used.")
```

Comment:
Similarly here, if its not used can this be removed?

Reply:
The class was deprecated as of 1.294 and tagged with the attribute 5 years ago. I'd be in favor of removing it, though possibly in a different PR.